lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -50,7 +62,7 @@
   %endif
 
     <div class="request-response msg msg-confirm copy" id="report-request-response"></div>
-    <div class="request-response-error msg msg-warning copy" id="report-request-response-error"></div>
+    <div class="request-response-error msg msg-error copy" id="report-request-response-error"></div>
```

Comment:
Problematically this div is now so low I don't see it when I click the button and I'm on my laptop. This is going to be the case for all the above buttons as well.

Would it be possible to add a handler that, for all button clicks in this section, brought you to this point on the page to highlight the message?